bin/live/pycbc_live_combine_single_fits

```diff
@@ -121,24 +125,28 @@ for ifo in args.ifos:
     fout.create_group(ifo)
     fout[ifo].attrs['live_time'] = sum(live_times[ifo])
 
+qupper = args.conservative_percentile / 100.
+qstring = ("%f" % args.conservative_percentile).rstrip('0').rstrip('.')
```

Comment:
This code seems a bit overcomplicated. I suggest forcing `--conservative-percentile` to take an integer argument, and using
```
qstring = str(args.conservative_percentile)
```
(or not introducing the `qstring` variable at all, in fact).